lib/model-builder.js

```diff
@@ -209,6 +211,11 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
 
   util.inherits(ModelClass, ModelBaseClass);
 
+  // inherit validations
+  if (ModelBaseClass.validations && !settings.realmRequired) {
+    ModelClass.validations = traverse(ModelBaseClass.validations).clone();
```

Comment:
If `validations` is an object, then I am proposing to use prototypal inheritance here:

```js
ModelClass.validations = Object.create(ModelBaseClass.validations);
```

If it's an array, then I think we can simply clone the array only?

```js
ModelClass.validations = ModelBaseClass.validations.slice();
```

Thoughts?